plugins/dev-create/bin/common.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs-extra');
+
+exports.createDir = function(path) {
```

Comment:
A name (and JSDoc) distinguishing how this is different from `fs.mkdir` would be useful.  Not sure exactly what name would be best; `createNewDir` maybe?